src/components/Clock.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  today: Date;
+};
+
+const getNormTime = (date: Date) => (
+  date.toUTCString().slice(-12, -4)
+);
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  clockId = 0;
+
+  componentDidMount() {
+    this.clockId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      window.console.info(
+        getNormTime(this.state.today),
+      );
```

Comment:
Don't use this state value here, it can be outdated
Save result of `new Date()` to constant and use it in this function